1.data-exploration/scripts/nbconverted/1.model_data_visualization.py

```diff
@@ -357,16 +345,20 @@
 
 
 # Adult solid vs liquid tumors
+cancer_types = adult_model_df['OncotreePrimaryDisease'].tolist()
+
+liquid = []
+non_liquid = []
+
+for type in cancer_types:
+    if liquid_tumors[0] in type or liquid_tumors[1] in type:
+        liquid.append(type)
+    else:
+        non_liquid.append(type)
+
```

Comment:
Wait, is this repeated? Oh, maybe the difference is `adult_model_df`? If so, please make the variable names even more specific :)

Reply:
Yep, renaming `non_liquid` and `liquid` to `adult_non_liquid` and `adult_liquid`

Reply:
And also changing `type` to `cancer_type` as in previous instance